deps/build.jl

```diff
@@ -9,7 +9,7 @@ else
 
     provides(Binaries, URI("http://bitbucket.org/MDukhan/yeppp/downloads/yeppp-$ver.tar.bz2"), libyepppdep, unpacked_dir = "yeppp-$ver/binaries/windows/amd64/", os = :Windows)
     provides(Binaries, URI("http://bitbucket.org/MDukhan/yeppp/downloads/yeppp-$ver.tar.bz2"), libyepppdep, unpacked_dir = "yeppp-$ver/binaries/macosx/x86_64/", os = :Darwin)
-    @linux_only push!(BinDeps.defaults, Binaries)
+    @compat @static is_linux() ? push!(BinDeps.defaults, Binaries) : nothing
```

Comment:
and shouldn't actually need `@compat` here, just `using Compat` at the top - but change the minimum required version of `Compat` in the `REQUIRE` file to the version that introduced `@static` and `is_linux`.